sdks/python/apache_beam/examples/fastavro_it_test.py

```diff
@@ -135,29 +146,9 @@ def batch_indices(start):
     assert result.state == PipelineState.DONE
 
     with TestPipeline(is_integration_test=True) as fastavro_read_pipeline:
-
-      fastavro_records = \
-          fastavro_read_pipeline \
-          | 'create-fastavro' >> Create(['%s*' % fastavro_output]) \
-          | 'read-fastavro' >> ReadAllFromAvro() \
-          | Map(lambda rec: (rec['number'], rec))
-
-      def check(elem):
-        v = elem[1]
-
-        def assertEqual(l, r):
-          if l != r:
-            raise BeamAssertException('Assertion failed: %s == %s' % (l, r))
-
-        assertEqual(sorted(v.keys()), ['fastavro'])
-        fastavro_values = v['fastavro']
-        assertEqual(len(fastavro_values), 1)
-
-      # pylint: disable=expression-not-assigned
-      {
-          'fastavro': fastavro_records
-      } \
-      | CoGroupByKey() \
+      fastavro_read_pipeline \
+      | 'create-fastavro' >> Create(['%s*' % fastavro_output]) \
+      | 'read-fastavro' >> ReadAllFromAvro() \
```

Comment:
Can we also compare the values for the keys, to make sure that no values were not lost during write-read operation? 
I think it could be accomplished by running co-GBK of a pcollection  coming form `     | 'read-fastavro' >> ReadAllFromAvro() \`, and  pcollection of generated data. Then, we can extract the set of elements tagged with first pcollection, and the second pcollection, and verify that these sets are the same for all elements in GBK output.

Reply:
I changed according to the above suggestions.